docker/.env.start.test.template

```diff
@@ -1,2 +1,4 @@
 # Disabling repo_downloader.sh
 REPO_DOWNLOADER_ENABLED=false
+
+LOCAL_ELASTICSEARCH=0
```

Comment:
I know that at code level is the same but why not use normal `false` and `true`? There is no reason to do not do it. Clarity is very important.

Reply:
Because the variable is read as a string and ```bool("false")``` is actually ```True```